app/utils.py

```diff
@@ -3,6 +3,9 @@
 import getpass
 import keyring
 import yaml
+import requests
+
+BACKEND_SERVER_URL = "http://localhost:8080/upload"
```

Comment:
One day this will be `neuralaunch.lainforge.org`

Reply:
ONE DAY